iot_packet_verifier/src/lib.rs

```diff
@@ -0,0 +1,133 @@
+use futures::stream::StreamExt;
+use slog::{self, info, o, Drain, Level, Logger};
+use std::collections::HashMap;
+//use std::io::{ErrorKind, Result};
+//use tokio_stream::StreamExt;
+// use tokio::sync::mpsc;
+
+use helium_proto::services::router::PacketRouterPacketReportV1;
+use helium_proto::Message;
+use poc_store::{file_source, Result};
+// FIXME for writing to S3:
+//use poc_store::FileStore;
+
+// Inputs:
+type GatewayPubKey = Vec<u8>;
+#[allow(clippy::upper_case_acronyms)]
+type OUI = u32;
+
+// Outputs:
+// type Bookkeeping = helium_proto::IotPacketVerifierBookkeeping;
+// type OuiBooks = helium_proto::OuiBookkeeping;
+// type GatewayBooks = helium_proto::GatewayBookkeeping;
+
+// While processing:
+
+pub struct Counts {
+    pub gateway: GatewayPubKey,
+    pub oui: OUI,
+}
+
+// TODO move to Settings.toml
+// Same var names as used in HPR's .env files.
+/*
+static AWS_ACCESS_KEY_ID: &str = "AWS_ACCESS_KEY_ID";
+static AWS_SECRET_ACCESS_KEY: &str = "AWS_SECRET_ACCESS_KEY";
+static AWS_DEFAULT_REGION: &str = "AWS_DEFAULT_REGION";
+static PACKET_REPORTER_BUCKET_NAME: &str = "PACKET_REPORTER_BUCKET_NAME";
+*/
+
+#[derive(Debug)]
+pub struct PacketCounters {
+    pub gateway: HashMap<GatewayPubKey, u32>,
+    pub oui: HashMap<OUI, u32>,
+}
+
+impl PacketCounters {
+    pub fn new() -> Self {
+        Self {
+            gateway: HashMap::new(),
+            oui: HashMap::new(),
+        }
+    }
+}
+
+impl Default for PacketCounters {
```

Comment:
I think you can derive `Default` for `PacketCounters`

Reply:
Thanks!  Fixed in commit b83dbea.